nbparameterise/code.py

```diff
@@ -35,10 +35,19 @@ def __eq__(self, other):
                 and self.type == other.type
                 and self.value == other.value
             )
+def findFirstParametersTaggedCell(nb):
```

Comment:
Let's stick to `snake_case` naming, like the other functions here. And there should be 1-2 blank lines before the new function (I'm not strict about following a particular code style, but I don't like having no gap at all :-)

Reply:
fixed